qemu/tests/usb_host.py

```diff
@@ -28,21 +38,24 @@ def usb_dev_hotplug():
 
     @error_context.context_aware
     def usb_dev_verify():
-        error_context.context("Check usb device [%s] in guest" % usb_options,
-                              logging.info)
+        error_context.context("Check usb device in guest", logging.info)
         session.cmd(lsusb_cmd)
 
     @error_context.context_aware
-    def usb_dev_unplug():
-        error_context.context("Unplug usb device", logging.info)
-        vm.monitor.cmd(monitor_del)
+    def usb_dev_unplug(dev):
+        error_context.context("Unplug usb-host device", logging.info)
+        session.cmd("dmesg -c")
+        vm.devices.simple_unplug(dev, vm.monitor)
         session.cmd_status("sleep 2")
         messages_del = session.cmd("dmesg -c")
         for line in messages_del.splitlines():
             logging.debug("[dmesg del] %s" % line)
         if messages_del.find(match_del) == -1:
             test.fail("kernel didn't detect unplug")
 
+    usb_params = {}
+    usb_params["bus"] = params["usb_bus"]
```

Comment:
This is not needed, the device can get the parent bus according to the type of `usb_controller`.

Reply:
Updated, thanks!